index.html

```diff
@@ -90,7 +103,7 @@ <h3 class="project-title project-info">Profesional Art<br>Printing Data</h3>
         <button type="button" class="project-button">See Project</button>
```

Comment:
This is quite minute and I know you can achieve it easily. Please, adjust the width of the project button to align with the width of the card. This will apply when the project is hovered over since that is when the button shows up.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206508121-8306ccce-eac5-4108-a24a-806bac7ca4be.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/206508376-41c3bf6f-d4c1-4151-899e-97f0de776b78.png)